File: src/sdk/php/3/core-classes/memory-storage/georadius/snippets/georadius-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $points = $kuzzle->memoryStorage()->georadius('key', 15, 37, 200, 'km');
}
catch (ErrorException $e) {

}

File: src/sdk/java/2/core-classes/memory-storage/georadius/snippets/georadius-1.java

kuzzle.memoryStorage.georadius("key", 15, 37, 200, "km", new ResponseListener<JSONObject[]>() {
  @Override
  public void onSuccess(JSONObject[] points) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
